cmd/shell.go

```diff
@@ -178,6 +180,11 @@ type Config struct {
 
 		DNSResolver string
 		DNSTimeout  string
+
```

Comment:
Why is this showing up in this PR?

You have another PR open that creates a PolicyDB struct, too, I think? Or something similar.